test/fixtures/test-runner/output/default_output.snapshot

```diff
@@ -0,0 +1,57 @@
+[32m✔ should pass [90m(*ms)[39m[39m
```

Comment:
Could we run these with ANSI colors support disabled to make easier-to-read snapshots?

Reply:
the entire purpose of this test is to test the colors and the reporter